lumicks/pylake/tests/test_file.py

```diff
@@ -102,6 +103,17 @@ def test_marker(h5_file):
         assert np.isclose(f.markers["test_marker2"].stop, 300)
 
 
+def test_marker_metadata(h5_file):
+    f = pylake.File.from_h5py(h5_file)
+
+    if f.format_version == 2:
+        assert np.isclose(f.markers["force feedback"].start, 200)
+        assert np.isclose(f.markers["force feedback"].stop, 300)
```

Comment:
ultimate nit-picky comment here, but I'd move these 2 lines to the end of this block .... keep all of the `markers["force feedback"]` grouped together

Reply:
sure :)